specs/protocol/holocene/exec-engine.md

```diff
@@ -62,6 +63,36 @@ field.
 | `denominator` | `u32 (big-endian)` | `[0, 4)`    |
 | `elasticity`  | `u32 (big-endian)` | `[4, 8)`    |
 
+### Extended `ExecutionPayloadV3`
+
```

Comment:
Does this mean there needs to be a new gossipsub topic for the modified execution payload? For unsafe blocks. Each version gets its own topic, see https://github.com/ethereum-optimism/specs/pull/396 which modifies the execution payload and a new topic is created

Reply:
Hmm, since it is just adding a new optional field and the block itself is not changed, I assumed no.  But I am not so familiar with how the p2p layer does validation and may have to dig deeper to confirm.